src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        List<Integer> list = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            list.add(i);
+        }
+        for (int i = 0; i < 5; i++) {
+            list.add(2);
+        }
+        System.out.println(list);
+        list.remove(Integer.valueOf(2));
```

Comment:
The removal of elements from the list using 'remove(Integer.valueOf(2))' is inefficient in a loop as it will always search for the element from the beginning of the list. Consider using a different approach if you want to remove multiple occurrences of the same element.